enterprise/validators.py

```diff
@@ -33,5 +33,5 @@ def validate_image_size(image):
     Validate that a particular image size.
     """
     config = get_app_config()
-    if config and not image.size < getattr(config, "image_size", 0):
-        raise ValidationError(_("The logo image file size must be less than 4KB."))
+    if config and not image.size <= getattr(config, "image_size", 0):
+        raise ValidationError(_("The logo image file size must be less than or equal to 512KB."))
```

Comment:
This string is wrong. In combination with the above change to store the max image size in KB, change this string to be formatable so that our configured value is what shows up as the error message, rather than just the platform default.